test/scripts/dbq.py

```
@@ -0,0 +1,21 @@
+conn = sqlite3.connect(DB)
+conn.row_factory = sqlite3.Row
+
+query = sys.argv[1] if len(sys.argv) > 1 else "SELECT 'hello'"
+try:
+    rows = conn.execute(query).fetchall()
+    if not rows:
+        print("(no rows)")
+    else:
+        for r in rows:
+            print(dict(r))
+except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)